docs/source/concept_models/vivarium_ciff_sam/concept_model.rst

```diff
@@ -149,13 +149,68 @@ Historically, prevention research has primarily focused on stunting, and, as a r
 ------------------------
 
 **Baseline**
-The baseline scenario will project GBD 2019 demographic and disease trends out from 2022 to 2027 and coverage rates for all preventive and therapeutic interventions will be held constant across the 5 years of the microsimulation to simulate a business-as-usual treatment scenario.
+The baseline scenario will project GBD 2019 demographic and disease trends and GBD 2020 exposure trends out from 2022 to 2027 and coverage rates for all preventive and therapeutic interventions will be held constant across the 5 years of the microsimulation to simulate a business-as-usual treatment scenario.
 
 **Alternative scenario 1**
-Scale up the coverage of MAM management and MAM treatment interventions to 90%
+Scale up the 'effective-coverage' of GAM treatment from baseline level of effective-coverage to scenario level of effective-coverage. Those who are 'effectively covered' have a shorter duration of SAM and MAM episodes. In this first alternative scenario, a larger proportion of simulants will be effectivey covered. Keeping incidence of MAM (i2) and SAM (i1) the same as baseline, we expect the prevalence of SAM and MAM to decrease  with a shorter duration (prevalence ~ incidence x duration).    
+
+| Effective coverage = treatment coverage x treatment efficacy
+| Not effectively covered = 1 - (treatment coverage x treatment efficacy)
+
+.. list-table:: Effective coverage of GAM treatment program
+  :widths: 10 10 10 15 15 15 20 
+  :header-rows: 1
+
+  * - Exposure 
+    - Treatment coverage
+    - Treatment efficacy
+    - Effectivey covered
+    - Not effectively covered
+    - Reference
+    - Note
+  * - SAM baseline 
+    - 48.8% (95% CI: 37.4, 60.4)
+    - 70% (95% CI: 64, 76)
+    - 0.488 x 0.7 = 0.34
+    - 1 - 0.34 = 0.66
+    - [Isanaka_etal_2021]_ , [Zw_2020]_
+    - This is for SAM-OTP which is ~98% of SAM.  
+  * - SAM alternative 
+    - 90%
+    - 75% 
+    - 0.9 x 0.75 = 0.675
+    - 1 - 0.675 = 0.325
+    - Sphere standards
+    - Sphere guideline for efficacy only 
+  * - MAM baseline 
+    - 48.8% (95% CI: 37.4, 60.4)
+    - 73.1% for RUSF
+    - 0.488 x 0.731 = 0.34
+    - 1 - 0.34 = 0.66
+    - [Ackatia_Armah_2015]_
+    - Baseline coverage of MAM needs to be updated, efficacy comes from trial and may be too optimistic
```

Comment:
Why does the 48.8% coverage need updating? What is it based on now? Does it also need to be updated for SAM?